package.json

```diff
@@ -28,6 +28,7 @@
   "author": "Connor Turland <connor@sprillow.com>",
   "license": "CAL-1.0",
   "dependencies": {
+    "d3-dag": "^0.11.5",
```

Comment:
add this dependency to the `web/package.json` instead

Reply:
closing as this has been fixed in a more recent commit.